storage/local/persistence.go

```diff
@@ -1173,11 +1173,12 @@ func (p *persistence) purgeArchivedMetric(fp clientmodel.Fingerprint) (err error
 
 // unarchiveMetric deletes an archived fingerprint and its metric, but (in
 // contrast to purgeArchivedMetric) does not un-index the metric.  If a metric
-// was actually deleted, the method returns true and the first time of the
-// deleted metric. The caller must have locked the fingerprint.
+// was actually deleted, the method returns true and the first time and last
+// time of the deleted metric. The caller must have locked the fingerprint.
 func (p *persistence) unarchiveMetric(fp clientmodel.Fingerprint) (
 	deletedAnything bool,
-	firstDeletedTime clientmodel.Timestamp,
+	firstTime clientmodel.Timestamp,
```

Comment:
I actually prefer having them aligned under each other for legibility. Not strongly, but will keep it as is for now :)